charts/et-sya-api/values.yaml

```diff
@@ -3,10 +3,18 @@ java:
   image: 'hmctspublic.azurecr.io/et/sya-api:latest'
   ingressHost: et-sya-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: et
-#  Uncomment once the vault containing the app insights key has been set up
-#  keyVaults:
-#    et:
-#      secrets:
-#        - name: AppInsightsInstrumentationKey
-#          alias: azure.application-insights.instrumentation-key
+  keyVaults:
+    et-sya-api:
+      secrets:
+        - name: microservicekey-et-sya-api
```

Comment:
It looks like this secret has been created manually. Really it should be created using terraform
https://github.com/hmcts/et-sya-api/blob/750ccb24064e01bb662aa0665ab504dbbbc53bc6/infrastructure/main.tf#L50

Reply:
Renamed values.yaml to fix this